test/types/file/unableToOpenFile.good

```diff
@@ -1 +1 @@
-unableToOpenFile.chpl:1: error: No such file or directory in open with path "_test_cannotOpenMe.txt"
+unableToOpenFile.chpl:1: error: uncaught error - No such file or directory in open with path "_test_cannotOpenMe.txt"
```

Comment:
The style like
```
unableToOpenFile.chpl:1: error: uncaught error - No such file or directory in open with path "_test_cannotOpenMe.txt"
```

seems redundant - maybe it could be:
```
unableToOpenFile.chpl:1: uncaught error: No such file or directory in open with path "_test_cannotOpenMe.txt"
```

On a related note, in an ideal world, creating an Error object will capture the file/line information of where the error was created (and possibly more of a stack trace). In that world, we'd probably have multiple lines output per error, since we'd want to report two (or more) locations. What would the ideal output look like in that case? 

In most of these examples, the compiler is intentionally hiding the distinction (by reporting the "user code location" ignoring internal modules). But we could easily construct a case where a user module creates a function that throws and then calls it. Should the linenumber reported be to where the error is created? Where it is thrown? Or where it is not caught? (I think it should show where it is created & where it is not caught).